### PR TITLE
Update ewok, solo, r2d2 packages with new releases

### LINKED ISSUE #123: Update solo, r2d2, ewok packages
Following the JCSDA Skylab release 1.0, update the `solo`, `r2d2`, `ewok` packages in the `jcsda-emc` repo (directory).

### PR DESCRIPTION
**Pull request by climbfuji**
Update solo, r2d2, ewok packages following the Skylab-1.0 release. Tested on Orion, Discover, AWS (Ubuntu, Red Hat), macOS, with Intel, GNU, clang+gfortran.

Fixes https://github.com/NOAA-EMC/spack/issues/123

### CODE REVIEW

File: var/spack/repos/jcsda-emc/packages/ewok/package.py [hunk 1]
```diff
@@ -17,7 +16,8 @@ class Ewok(PythonPackage):
     maintainers = ['climbfuji', 'ericlingerfelt']
 
     version('develop', branch='develop', no_cache=True)
-    version('0.0.1', commit='69fff0f460fdb639db4fd38574dee8262b8a1f84', preferred=True)
+    version('0.0.1', commit='69fff0f460fdb639db4fd38574dee8262b8a1f84')
+    version('0.1.0', commit='2f88d2ea99bc88a16154520f69f8d70d4aedafae', preferred=True)
```
Comment (kgerheiser) [inline var/spack/repos/jcsda-emc/packages/ewok/package.py]: Higher version should be on top

Comment (climbfuji) [reply var/spack/repos/jcsda-emc/packages/ewok/package.py]: Thanks, fixed.

**Review by kgerheiser — APPROVED** [review]

**Comment by kgerheiser** [comment]
I want to see what these tests do before merging

**Comment by climbfuji** [comment]
> I want to see what these tests do before merging

Was thinking the same.

**Comment by kgerheiser** [comment]
Well py-pyhdf is still an issue. I don't completely understand why this is an issue. Spack compares to `GITHUB_BASE_REF` for this.

**Comment by kgerheiser** [comment]
And gsl-lite has some placeholder text still.

I can continue to iterate on this.

**Pull request merged**